sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -715,6 +715,62 @@ public virtual Response<MetricAnomalyDetectionConfiguration> GetMetricAnomalyDet
             }
         }
 
+        /// <summary>
+        /// </summary>
```

Comment:
not that docstrings are coming, should this be added here too?

Reply:
docstrings will be added in a follow-up PR